### PR TITLE
[WIP] init(config): add prom-k8s config [OSD-6310]

### PR DESCRIPTION
**Pull request by georgettica**
as the resources do not exist by default, they need to be added

### CODE REVIEW

**Comment by georgettica** [comment]
`make generate-syncset` fails with the error
```
Error: obj '{"apiVersion": "rbac.authorization.k8s.io/v1", "kind": "RoleBinding", "metadata": {
    "name": "prometheus-k8s", "namespace": "openshift-rbac-permissions"}, "roleRef": {
    "kind": "Role", "name": "prometheus-k8s"}, "subjects": [{"kind": "ServiceAccount",
      "name": "prometheus-k8s", "namespace": "openshift-monitoring"}]}
' at path 'roleRef/name': apiGroup cannot be found in roleRef {"kind": "Role", "name": "prometheus-k8s"}
```

File: config/prom-k8s/prom-k8s-role.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: openshift-rbac-permissions
```
Comment (RiRa12621) [inline config/prom-k8s/prom-k8s-role.yaml:5]:
```suggestion
  namespace: openshift-route-monitor-operator
```

File: config/prom-k8s/prom-k8s-rolebinding.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: prometheus-k8s
+  namespace: openshift-rbac-permissions
```
Comment (RiRa12621) [inline config/prom-k8s/prom-k8s-rolebinding.yaml:5]:
```suggestion
  namespace: openshift-route-monitor-operator
```

File: config/prom-k8s/prom-k8s-role.yaml (hunk 1 above)
Comment (georgettica) [reply config/prom-k8s/prom-k8s-role.yaml:5]: oh, I didn't think of it, will parameterize it :)

File: config/prom-k8s/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+resources:
+- prom-k8s-role.yaml
+- prom-k8s-rolebinding.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: openshift-rbac-permissions
```
Comment (RiRa12621) [inline config/prom-k8s/kustomization.yaml:6]: namespace: openshift-route-monitor-operator

**Comment by georgettica** [comment]
the PR now creates the packagemanifest, but I am not sure if it actually works in the real world

**Comment by georgettica** [comment]
Made the PR create a package manifest, now we just need to test it :)

**Pull request closed without merging**